### PR TITLE
Update and refactor ES6 syntax to improve readability and clean.

### PR DESCRIPTION
**Pull request by wenkesj**
# ES6 Refactor and Prune

A few things I noticed were that the exporting of some classes weren't uniform and the use of a statistical analyzer was just a class with methods and no data. I decided a few functional improvements to this was to facilitate a better looking and less bulky source.
- Ssim class -> utility functions
- Short hand default exports
- Export all (*) for main file
- Update english on the readme

### CODE REVIEW

**Comment by wenkesj** [comment]
@austinkelleher @xdumaine

File: examples/troubleshooter/test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ let video = true;
 let audio = true;
 
 let iceServers = [];
-const webRTCTroubleshooter = WebRTCTroubleshooter.default;
```
Comment (xdumaine) [inline examples/troubleshooter/test.js]: I'm confused about what happened here to make this work without `default`

Comment (wenkesj) [inline examples/troubleshooter/test.js]: In the `src/index.js` reference that we are `require`ing, I chose not to export the module as default. Babel 6.x treats the `exports` object when `export default ...` as export["default"]. I chose to collectively export all the modules with `export *`.

**Comment by wenkesj** [comment]
Seems to be referencing [this.statistics -> statistics](https://github.com/MyPureCloud/webrtc-troubleshooter/pull/1/commits/006b7f528167cfcf587c7dda608b41dd686a11fe#diff-c52d9e6ba681b0858d307275a3f7a8dbR60) when I trampled that commit. But I think it's interesting to see this module evolve from where it started!

**Comment by xdumaine** [comment]
@wenkesj would you mind reverting the bundle file and the version bump to package.json? I'll get those after merge when I publish.

**Pull request closed without merging**